Source/Urho3D/Graphics/Graphics.h

```diff
@@ -795,6 +799,8 @@ class URHO3D_API Graphics : public Object
     String orientations_;
     /// Graphics API name.
     String apiName_;
+    String rendererName_;
+    String versionString_;

```

Comment:
These do not have doc comments. Yes, i realize they will be redundant. But for the sake of consistency..